lib/components/viewers/pattern-row.tsx

```diff
@@ -101,6 +103,11 @@ const PatternRow = ({
             />
           </span>
           <span style={{ wordBreak: 'break-word' }} title={pattern.headsign}>
+            {alwaysShowLongName && (
```

Comment:
```suggestion
            {alwaysShowLongName && pattern.route.longName && (
```

Wouldn't this prevent the threat of a crash??

Reply:
It would. My thinking was that this would discourage people from using the feature when it didn't make sense, but I think I agree with you. Will leave it to the second reviewer to decide?